eclair-core/src/main/scala/fr/acinq/eclair/io/Monitoring.scala

```diff
@@ -21,8 +21,10 @@ import kamon.Kamon
 object Monitoring {
 
   object Metrics {
-    val PeerConnections = Kamon.counter("peers.connecting.count")
-    val ConnectedPeers = Kamon.gauge("peers.connected").withoutTags()
+    val PeerConnectionConnecting = Kamon.counter("peerconnection.connecting")
+    val PeerConnectionConnected = Kamon.gauge("peerconnection.connected")
```

Comment:
Are you using this one to monitor for discrepancies with `PeerConnected`?

Reply:
Agreed, what is the point of this specific gauge ?

Reply:
Our code guarantees that we have one single instance of `Peer` for a given `remoteNodeId`, but we can have several `PeerConnection` to the same `remoteNodeId` even if that should be a transient state. This is particularly worh monitoring in a distributed deployment.

Does that make sense?

Reply:
Agreed, it's worth verifying that we don't constantly have much more `peerconnection.connected` than `peer.connected`